### PR TITLE
Fix for #3094

### PR DESCRIPTION
**Pull request by Shqrdx**
Looks like MacGamePlatform misses the Present overload and metrics will not recreated. I dont have Mac so I dont know perfectly will it works or not.

### CODE REVIEW

**Comment by Shqrdx** [comment]
I mean this - https://github.com/mono/MonoGame/issues/3904 sorry for confusion

**Comment by Shqrdx** [comment]
I am not sure if calling full Present would be good to other parts of rendering so I change it to raw recreation of metrics.

File: MonoGame.Framework/Desktop/OpenTKGamePlatform.cs [hunk 1]
```diff
@@ -283,8 +283,6 @@ public override void Log(string Message)
 
         public override void Present()
         {
-            base.Present();
```
Comment (tomspilman) [inline MonoGame.Framework/Desktop/OpenTKGamePlatform.cs]: Why was this removed?

File: MonoGame.Framework/MacOS/MacGamePlatform.cs [hunk 2]
```diff
@@ -380,6 +380,13 @@ public override void ResetElapsedTime()
         {
             _gameWindow.ResetElapsedTime();
         }
+		
+		public override void Present()
+        {
+            var device = Game.GraphicsDevice;
+            if (device != null)
+                device._graphicsMetrics = new GraphicsMetrics();
```
Comment (tomspilman) [inline MonoGame.Framework/MacOS/MacGamePlatform.cs]: This is pretty gross... but we plan to kill the `MacGamePlatform` in favor of `OpenTKGamePlatform` soon anyway.

File: MonoGame.Framework/Desktop/OpenTKGamePlatform.cs (hunk 1 above)
Comment (Shqrdx) [inline MonoGame.Framework/Desktop/OpenTKGamePlatform.cs]: Because it is a call of empty method. I recommend either mark Present in GamePlatform abstract or place other content from Present of OpenTKGamePlatform to GamePlatform itself.

Comment (tomspilman) [inline MonoGame.Framework/Desktop/OpenTKGamePlatform.cs]: Yeah... it should be made abstract and all platforms should implement it.

File: MonoGame.Framework/MacOS/MacGamePlatform.cs (hunk 2 above)
Comment (Shqrdx) [inline MonoGame.Framework/MacOS/MacGamePlatform.cs]: Than this issue resolve without my PR, because OpenTKGamePlatform already have Present.

Comment (Shqrdx) [inline MonoGame.Framework/MacOS/MacGamePlatform.cs]: As I said 
>I am not sure if calling full Present would be good to other parts of rendering on Mac so I change it to raw recreation of metrics.

So its gross, yeah..

Comment (KonajuGames) [inline MonoGame.Framework/MacOS/MacGamePlatform.cs]: Can you explain what you mean by this?  Why would it not be good?

Comment (tomspilman) [inline MonoGame.Framework/MacOS/MacGamePlatform.cs]: I think all it means is that the Mac code didn't do it before... so changing it might break something.

Comment (Shqrdx) [inline MonoGame.Framework/MacOS/MacGamePlatform.cs]: > I think all it means is that the Mac code didn't do it before... so changing it might break something.

Exactly

**Comment by Shqrdx** [comment]
@tomspilman @KonajuGames I guess it is not good for merging ?

**Comment by KonajuGames** [comment]
I'll have a closer look when I get home.

**Comment by kjpou1** [comment]
Sorry guys.  Will make the time to verify this tonight.

**Comment by kjpou1** [comment]
@Shqrdx @tomspilman @KonajuGames Verified that this is working.  Thanks.

**Comment by KonajuGames** [comment]
Thanks @kjpou1 and @Shqrdx . Merging.
